packages/python-pip/build.sh

```diff
@@ -3,11 +3,12 @@ TERMUX_PKG_DESCRIPTION="The PyPA recommended tool for installing Python packages
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="24.0"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=2
 TERMUX_PKG_SRCURL=https://github.com/pypa/pip/archive/$TERMUX_PKG_VERSION.tar.gz
 TERMUX_PKG_SHA256=ad0dfe75fb28092a8cbe18523391695ceb0c0d65a5c9a969349fcb13b12b84c7
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_TAG_TYPE="newest-tag"
+TERMUX_PKG_UPDATE_VERSION_REGEXP='\d+\.\d+\.\d+'
```

Comment:
Emmm... This variable may be misused here. `TERMUX_PKG_UPDATE_VERSION_REGEXP` (and `TERMUX_PKG_UPDATE_VERSION_SED_REGEXP`) is used to extract a valid version string from tags.

For example, the build script of [ngspice](https://github.com/termux/termux-packages/blob/c23a1881062fc644e94d4666179eb564de883e73/packages/ngspice/build.sh#L11) uses it because the tag gotten from `termux_github_api_get_tag` is like `ngspice-42`, but only some part of it (`42`) should be used as the package version string.

You can override `termux_pkg_auto_update` using something like (untested):
```
termux_pkg_auto_update() {
	local latest_tag="$(termux_github_api_get_tag "${TERMUX_PKG_SRCURL}" "${TERMUX_PKG_UPDATE_TAG_TYPE}")"
	local filtered_latest_tag="$(grep -oP "${TERMUX_PKG_UPDATE_VERSION_REGEXP}" <<< "${LATEST_VERSION}" || true)"
	if [[ -z "${filtered_latest_tag}" ]]; then
		echo "INFO: No update needed, tag '${latest_tag}' is not valid." 
	fi
}
```

Reply:
Ah okay so I need to override the update function, okay thanks.

Reply:
Thanks for the assist, I was under the impression that `TERMUX_PKG_UPDATE_VERSION_REGEXP` would cause it to consider it as "no update needed", in hindsight that obviously doesn't make much sense.